packages/graphql-typescript-definitions/src/print/document/utilities.ts

```diff
@@ -1,5 +1,5 @@
 import {upperCaseFirst} from 'upper-case-first';
-/* eslint-disable @typescript-eslint/ban-ts-ignore */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
```

Comment:
I think we can change the `@ts-ignore` linnes to be `@ts-expect-error TS needs this implict reference though we never use it explicitly` then we can remove these eslint-disable lines